### PR TITLE
DTSPB-2932 CW Default dateOfDeathType on diedOn

### PR DESCRIPTION
**Pull request by MarcusHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-2932

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java [hunk 1]
```diff
@@ -643,7 +644,13 @@ public void verifyCaseworkerCreatedSolicitorApplicationTcSchema_TrustCorps() thr
 
         validatePostSuccessForPayload(payload, PAPER_FORM_URL);
     }
-    
+
+    @Test
+    public void verifyCaseworkerDefaultDateOfDeathType() throws IOException {
+        //String payload = getJsonFromFile("caseworkerDefaultDateOfDeathType.json");
```
Comment (DelythJustice) [inline src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java]: //

File: ccdImports/configFiles/CCD_Probate_Backoffice/CaseEvent.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "applyforGrantPaperApplicationMan", "Name": "PA1P/PA1A/Solicitors Manual", "Description": "Manual input to create applicantion", "DisplayOrder": 1, "PostConditionState": "CaseCreated", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/case/paperForm", "SecurityClassification": "Public", "ShowSummary": "Y"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "applyforGrantPaperApplicationMan", "Name": "PA1P/PA1A/Solicitors Manual", "Description": "Manual input to create applicantion", "DisplayOrder": 1, "PostConditionState": "CaseCreated", "CallBackURLAboutToStartEvent":  "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/case/initPaperForm", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/case/paperForm", "SecurityClassification": "Public", "ShowSummary": "Y"},
```
Comment (DelythJustice) [inline ccdImports/configFiles/CCD_Probate_Backoffice/CaseEvent.json:2]: I duno if you missed the dev advice from Sanjay on the story - CCD have added a defaultvalue abillity. Did you try it?

Comment (MarcusHMCTS) [reply ccdImports/configFiles/CCD_Probate_Backoffice/CaseEvent.json:2]: Yes, i've tried  the "DefaultValue" , but not working for the this

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java (hunk 1 above)
Comment (MarcusHMCTS) [reply src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java]: deleted,thx

**Review by DelythJustice — APPROVED** [review]

**Pull request merged**